README.md

```diff
@@ -26,12 +26,12 @@ $ yarn add @chuva.io/sisp
 
 ### Sisp with 3DSecure
 
-Import `@chuva/sisp/sisp3DS` and create a new instance using your credentials. (obtained from SISP):
+Import `@chuva.io/sisp/src/sisp3DS` and create a new instance using your credentials. (obtained from SISP):
```

Comment:
@luis737 We should not import from `/src`. The path was correct as it was. Please update the package so we can import it properly.

Reply:
I think it can be `const Sisp = require('@chuva.io/sisp3DS');`

Reply:
I agree